src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -9,13 +9,12 @@ import ch.qos.logback.core.FileAppender
 import com.typesafe.config.{Config, ConfigFactory}
 import cromwell.binding._
 import cromwell.binding.values.WdlFile
-import cromwell.engine.backend.Backend
+import cromwell.engine.backend.{CromwellBackend, Backend}
 import cromwell.engine.backend.jes.JesBackend
 import cromwell.engine.io.IoManager
 import cromwell.engine.io.gcs.GoogleCloudStorage
 import cromwell.engine.io.shared.SharedFileSystemIoInterface
 import cromwell.engine.workflow.WorkflowOptions
-import cromwell.server.CromwellServer
 import cromwell.util.TryUtil
```

Comment:
But is it worth mentioning that IntelliJ has no idea what's going on in the middle of this file? I don't know how far I'd trust it's auto-wizardry here